core/kazoo_endpoint/Makefile

```diff
@@ -9,5 +9,8 @@ test: kze-test
 
 kze-test:
 	$(MAKE) compile-test -C $(ROOT)/core/kazoo_number_manager/
+	$(MAKE) compile-test -C $(ROOT)/core/kazoo_sip/
+	$(MAKE) compile-test -C $(ROOT)/core/kazoo_stdlib/
+	$(MAKE) compile-test -C $(ROOT)/core/kazoo_schemas/
```

Comment:
This target is misleading. https://github.com/2600hz/kazoo/blob/master/doc/installation.md#longer-version says to run `make compile-test` from the root of the repo. Tracking this app's test dependencies here is bound to break.